### PR TITLE
2-10 Updated some files for category/group restrictions in admin page.

### PR DESCRIPTION
**Pull request by cfwdall**
Kindly also run this query on your psql.
insert into faqconfig (config_name ,config_value) values ('main.enable_category_restrictions',true);

thanks

### CODE REVIEW

File: phpmyfaq/admin/category.add.php [hunk 1]
```diff
@@ -37,6 +37,14 @@
         <div class="row">
             <div class="col-lg-12">
 <?php
+$current_user_id = 1;
```
Comment (thorsten) [inline phpmyfaq/admin/category.add.php]: Why do you set the ID to 1? Please use camelCase as well.

File: phpmyfaq/admin/category.edit.php [hunk 2]
```diff
@@ -24,6 +24,11 @@
     exit();
 }
 
+$current_user_id = 1;
```
Comment (thorsten) [inline phpmyfaq/admin/category.edit.php]: Why do you set the ID to 1? Please use camelCase as well.

File: phpmyfaq/admin/category.main.php [hunk 3]
```diff
@@ -285,7 +285,17 @@
     if (isset($category)) {
         unset($category);
     }
+    // old code
     $category = new PMF_Category($faqConfig, [], false);
+    //
+
+    // new code
+    if ($faqConfig->config['main.enable_category_restrictions'] == 'true' && $user->getUserId() != 1){
```
Comment (thorsten) [inline phpmyfaq/admin/category.main.php]: Why the check for user ID 1?

File: phpmyfaq/admin/record.edit.php [hunk 4]
```diff
@@ -26,9 +26,21 @@
     exit();
 }
 
+$current_user_id = 1;
```
Comment (thorsten) [inline phpmyfaq/admin/record.edit.php]: Why do you set the ID to 1? Please use camelCase as well.

File: phpmyfaq/admin/category.add.php (hunk 1 above)
Comment (cfwdall) [reply phpmyfaq/admin/category.add.php]: This is just to keep the the old view wherein when you're given an admin access, you'll be able to view other categories/group even if you're not tagged under it.

**Comment by cfwdall** [comment]
phpmyfaq/inc/PMF/Category.php
phpmyfaq/inc/PMF/Faq.php
phpmyfaq/inc/PMF/Perm/Medium.php

These files does not exist on the 2.10 branch yet. Kindly let me know if I'm doing something wrong. Thanks.

**Comment by thorsten** [comment]
Ah, the directory inc/ is now arc/

**Comment by cfwdall** [comment]
I see, will create a new merge request again with the suggestions you've made. Thanks!

**Comment by thorsten** [comment]
I thank you for your help!

**Comment by cfwdall** [comment]
And thank you too for pointing out my mistakes.

**Pull request closed without merging**